modules/admin-ui/src/main/webapp/scripts/shared/directives/editableMultiSelectDirective.js

```diff
@@ -71,8 +71,9 @@ angular.module('adminNg.directives')
         });
       };
 
-      scope.onBlur = function (event) {
+      scope.onBlur = function () {
         if (!scope.removedValue) {
+          scope.addCurrentValue();
```

Comment:
@KatrinIhler Just realised that this is a bad idea: the scope.AddCurrentValue() on blur causes the value to be stored. Problem: When navigating through the fields using the tab key, this will store the existing values (even if the user has not edited it at all) which creates a new version of the media package.

... we should fix this before 6.2 is released... do you have time?

Reply:
@staubesv I can look into it. So I guess the solution is to only save if the values have changed, right?

Reply:
Yes, similar to editSingleValueDirective which keeps track of the original value.

Reply:
Patch is out: #644